### PR TITLE
Mtracy/nav bar component

### PR DESCRIPTION
**Pull request by mrtracy**
Two commits:
- Converted title bar of website to be a mithril component
- Compress CSS Output from Styl

### CODE REVIEW

**Comment by tamird** [comment]
LGTM

**Comment by tamird** [comment]
Too bad TS isn't as easy as stylus: https://github.com/Microsoft/TypeScript/issues/8

File: ui/ts/pages/navigation.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          {
+            title: "Nodes",
+            route: "/nodes"
+          },
+          {
+            title: "Stores",
+            route: "/stores"
+          },
+          {
+            title: "Logs",
```
Comment (BramGruneir) [inline ui/ts/pages/navigation.ts]: Remove logs.  There's no need for it as a top level component.

File: ui/ts/typings/mithriljs/mithril.d.ts [hunk 2]
```diff
@@ -100,14 +100,18 @@ declare module _mithril {
...
 	interface MithrilComponent<T extends MithrilController> {
 		controller: MithrilControllerFunction|{ new(): T };
 		view: MithrilView<T>;
 	}
 
 	interface ParameterizedMithrilComponent<T extends MithrilController, Targ> {
-        controller: (x?:Targ)=>T
-		view: MithrilView<T>;
+        controller: (x?:Targ)=>T;
+		view: ParameterizedMithrilView<T, Targ>;
```
Comment (BramGruneir) [inline ui/ts/typings/mithriljs/mithril.d.ts]: can you re-space this whole file?

**Comment by BramGruneir** [comment]
LGTM

File: ui/ts/typings/mithriljs/mithril.d.ts (hunk 2 above)
Comment (mrtracy) [inline ui/ts/typings/mithriljs/mithril.d.ts]: This is our typing file, it's third party.  What I really need to do is port these changes back.

Comment (mrtracy) [inline ui/ts/typings/mithriljs/mithril.d.ts]: What I will do is make this match the local spacing.

**Pull request merged**